.circleci/config.yml

```diff
@@ -1237,7 +1237,7 @@ jobs:
       # Notification step
       - notify-failures-on-develop:
           channel: "C03N11M0BBN"  # #notify-ci-failures
-          mentions: "@protocol-devx-pod"
+          mentions: "@stefano" # temporarily just mention Stefano until the acceptance tests are stable (was: @protocol-devx-pod)
```

Comment:
Are you confident the mention of `@protocol-devx-pod` won't trigger a slack notification to that handle?

Reply:
I don't think the @protocol-devx-pod will make it's way into the job/function as it's commented out.
Or have I misunderstood you?

Reply:
No you're correct. I didn't see it was in a comment initially